en/docs/tutorials/scenarios/scenario4-user-signup-approval-flow.md

```diff
@@ -1,4 +1,72 @@
-# Scenario 4 - User Signup Approval Flow
+# Scenario 4 - Signing up a new user
+
+## User Story
+
+**Quantis allows external users to access their APIs. To take the full benefit out of these APIs, they have decided to open the registration to outside users since it can be a burden to enter all the details for every single user/consumer. Therefore, they request an approval workflow for user registration in such a way that the users will be registered to the system when a user with administrative privileges approves their registration after a manual validation of the user.  By allowing external parties to register to their system under a supervision, Quantis expects higher revenue in future. **
```

Comment:
```suggestion
Quantis allows external users to access their APIs. To take the full benefit out of these APIs, they have decided to open the registration to outside users since it can be a burden to enter all the details for every single user/consumer. Therefore, they request an approval workflow for user registration in such a way that the users will be registered to the system when a user with administrative privileges approves their registration after a manual validation of the user.  By allowing external parties to register to their system under a supervision, Quantis expects higher revenue in future. 
```

Reply:
Fixed